doc/oper.xml

```diff
@@ -1395,6 +1395,38 @@ gap> VerticesReachableFrom(D, 5);
 ]]></Example>
   </Description>
 </ManSection>
+
+<ManSection>
+  <Oper Name="VerticesReachableFrom" Arg="digraph, root"/>
+  <Returns>A list.</Returns>
+  <Description>
+    This operation returns a list of the vertices  <A>v</A>, for which there exists
+    a non-trivial directed walk from any of the provided <A>root</A>s to vertex <A>v</A> in the digraph
+    <A>digraph</A>. See Section <Ref Subsect="Definitions" Style="Number" />
+    for the definition of a non-trivial directed walk.
+    <P/>
```

Comment:
The manual entry should include some indication of what happens if the list of roots contains values that are not vertices of `digraph`.

Reply:
Good point - thanks! Resolved